linters/eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
+    "rules": {
+        "indent": [
+            2,
+            4
+        ],
+        "quotes": [
+            2,
+            "single"
+        ],
+        "linebreak-style": [
+            2,
+            "unix"
+        ],
+        "semi": [
+            2,
+            "always"
+        ],
+        "max-len": [2, 80, 4],
+        "no-unused-vars": [2, {"vars": "all", "args": "none"}],
+        "comma-dangle": [0],
+        "indent": [0],
+        "no-console": [0]
+    },
+    "env": {
+        "es6": true,
+        "node": true,
+        "browser": true,
```

Comment:
Oh, right. Back when we did the original, we thought we'd share it with the
front and back ends. But I don't think we want to do that anymore.

On Thu, Dec 24, 2015 at 6:42 PM Carter Bancroft <notifications@github.com>
wrote:

> In linters/eslintrc.json
> <https://github.com/aboutdotme/javascript/pull/2#discussion_r48430707>:
>
> > +            "unix"
> > +        ],
> > +        "semi": [
> > +            2,
> > +            "always"
> > +        ],
> > +        "max-len": [2, 80, 4],
> > +        "no-unused-vars": [2, {"vars": "all", "args": "none"}],
> > +        "comma-dangle": [0],
> > +        "indent": [0],
> > +        "no-console": [0]
> > +    },
> > +    "env": {
> > +        "es6": true,
> > +        "node": true,
> > +        "browser": true,
>
> Agreed, but I was just copying what was done for our old jshintrc. I
> didn't know what the history was. If there's not compelling reason I'll
> make them false.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aboutdotme/javascript/pull/2/files#r48430707>.
>